tests/cypress/e2e/api_helpers.ts

```diff
@@ -0,0 +1,138 @@
+export function createCmkAutomationUser(cmkUser: string, cmkPassword: string) {
```

Comment:
Am thinking about moving these to custom Cypress commands as well. Should be done if we ever need the return value of one of these calls.